config/formats.js

```diff
@@ -554,6 +554,39 @@ let Formats = [
 		banlist: ['Blacephalon', 'Kartana', 'Komala', 'Kyurem-Black', 'Porygon-Z', 'Silvally', 'Tapu Koko', 'Tapu Lele', 'Aerodactylite', 'King\'s Rock', 'Metagrossite', 'Razor Fang'],
 		restrictedMoves: ['Acupressure', 'Belly Drum', 'Chatter', 'Geomancy', 'Lovely Kiss', 'Shell Smash', 'Shift Gear', 'Spore', 'Thousand Arrows'],
 	},
+	{
+		name: '[Gen 7] Metronome Battle',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Metronome Battle</a>`,
```

Comment:
This thread has the old rules listed, as well as the "Shitmons" name.

Reply:
It's that or no link at all, there's no post that officially supports this on Smogon.

Also, old rules? These rules and those listed on that site should be equal to the best of my knowledge.

Reply:
> remove some of the dumb rules like level 666